From 238844aa967747d9d11cca0b94584393ee2b3083 Mon Sep 17 00:00:00 2001
From: Jesse Van Gavere <jesse.vangavere@teledyne.com>
Date: Thu, 16 Jul 2026 08:23:24 +0200
Subject: [PATCH 05200/15333] overlayfs.rst: remove mention of workdir needing
 to be empty

This requirement has not been true since v4.8 when automatic cleanup was
added in commit eea2fb4851e9 ("ovl: proper cleanup of workdir")

Signed-off-by: Jesse Van Gavere <jesse.vangavere@teledyne.com>
Reviewed-by: Amir Goldstein <amir73il@gmail.com>
Signed-off-by: Jonathan Corbet <corbet@lwn.net>
Message-ID: <20260716-overlayfs-rst-update-v2-1-ba25393ea7a3@teledyne.com>
---
 Documentation/filesystems/overlayfs.rst | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/Documentation/filesystems/overlayfs.rst b/Documentation/filesystems/overlayfs.rst
index eb846518e6ac..ba41c0694e62 100644
--- a/Documentation/filesystems/overlayfs.rst
+++ b/Documentation/filesystems/overlayfs.rst
@@ -123,8 +123,7 @@ At mount time, the two directories given as mount options "lowerdir" and
   mount -t overlay overlay -olowerdir=/lower,upperdir=/upper,\
   workdir=/work /merged
 
-The "workdir" needs to be an empty directory on the same filesystem
-as upperdir.
+The "workdir" needs to be a directory on the same filesystem as upperdir.
 
 Then whenever a lookup is requested in such a merged directory, the
 lookup is performed in each actual directory and the combined result
-- 
2.55.0

